changelog.txt

```diff
@@ -1,3 +1,12 @@
+0.1.1-preview
+============
+New Features:
+**Microsoft Identity Web now surfaces the ClaimConstants class**. This allows developers to build a unique ClaimsPrincipal. [See issue for more details](https://github.com/AzureAD/microsoft-identity-web/issues/100)
+
+Bug Fixes:
+**`AddSignIn()` now provides a more robust processing of authorities accepting them to end in `/` or not**. [See issue for more details](https://github.com/AzureAD/microsoft-identity-web/issues/19)
+**Setting the `ValidAudiences` in `AddProtectedWebApi()` now accepts `api://<guid>` or `https://<tenantDomain>`**. [See issue for more details](https://github.com/AzureAD/microsoft-identity-web/issues/52)
```

Comment:
Rephrase, perhaps? `ValidAudiences` accepts any custom audience (any string), not specifically those.

Reply:
good idea. thx.